### PR TITLE
Github Deployments: Show no results on deployments page when filter data is empty

### PR DESCRIPTION
**Pull request by vykes-mac**
Related to #

## Proposed Changes

* Add `NoRsults` component similar to what is shown when searching for repositories on repository table.

## Testing Instructions

* On deployments page search a repo that doesn't exist you should see the component

**Before**
image

**After**
image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/github-deployments/deployments/no-results.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { __ } from '@wordpress/i18n';
+
+export const NoResults = () => {
+	return (
+		<div className="github-deployments-list__no-results">
```
Comment (Aurorum) [inline client/my-sites/github-deployments/deployments/no-results.tsx:5]: What do you think about using the EmptyContent component like this and then overriding styles as necessary?

C1DDBF84-810E-4D61-BED8-DD802A3D3441
F90CEE54-89CF-4F44-BC62-ACD82BBA7A99

File: client/my-sites/github-deployments/deployments/no-results.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+			<p>{ __( 'No Results found' ) }</p>
```
Comment (Aurorum) [inline client/my-sites/github-deployments/deployments/no-results.tsx]: typo maybe?

```suggestion
			<p>{ __( 'No results found' ) }</p>
```

**Comment by zaguiini** [comment]
Thanks! Seems like a good compromise until Javier gets back.

File: client/my-sites/github-deployments/deployments/no-results.tsx (hunk 1 above)
Comment (zaguiini) [reply client/my-sites/github-deployments/deployments/no-results.tsx:5]: These are using the old guidelines and we will probably move to something newer in the future. I think a simple copy is good for now.

**Review by zaguiini — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/11487423

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @vykes-mac for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/11487423).

**Pull request merged**